src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue22443.cs

```diff
@@ -0,0 +1,25 @@
+﻿using NUnit.Framework;
+using UITest.Appium;
+using UITest.Core;
+
+namespace Microsoft.Maui.TestCases.Tests.Issues;
+
+public class Issue22443 : _IssuesUITest
+{
+    public Issue22443(TestDevice device)
+		: base(device)
+	{ }
+
+    public override string Issue => "App Crash on Scroll Animation while navigating away from Page";
+
+    [Test]
+	public void NoExceptionShouldBeThrown()
```

Comment:
Test is failing on catalyst

Reply:
<img width="905" alt="Screenshot 2024-05-19 at 00 57 28" src="https://github.com/dotnet/maui/assets/42434498/4724844b-9df4-4043-82e3-479fd261a068">
Should work now :)